test/jdk/ProblemList.txt

```diff
@@ -890,3 +887,33 @@ jdk/jfr/event/os/TestThreadContextSwitches.java                 8247776 windows-
 tools/jpackage/share/jdk/jpackage/tests/ModulePathTest3.java#id0    8248418 generic-all
 
 ############################################################################
+
+# Client manual tests
+
+java/awt/event/MouseEvent/SpuriousExitEnter/SpuriousExitEnter_1.java 7131438,8022539 generic-all
+java/awt/event/MouseEvent/SpuriousExitEnter/SpuriousExitEnter_2.java 7131438,8022539 generic-all
+java/awt/Modal/WsDisabledStyle/CloseBlocker/CloseBlocker.java 7187741 linux-all,macosx-all
+java/awt/Component/UpdatingBootTime/UpdatingBootTime.html 7194219 linux-all
+java/awt/xembed/server/TestXEmbedServerJava.java 8001150,8004031 generic-all
+java/awt/TextArea/MouseOverScrollbarWhenTyping/Test1.java 8005970 macosx-all,windows-all
+javax/swing/JFileChooser/6698013/bug6698013.java 8024419 macosx-all
+javax/swing/JColorChooser/8065098/bug8065098.java 8065647 macosx-all
+java/awt/Modal/PrintDialogsTest/PrintDialogsTest.java 8068378 generic-all
+java/awt/dnd/DnDFileGroupDescriptor/DnDFileGroupDescriptor.html 8080185 macosx-all,linux-all
+javax/swing/JTabbedPane/4666224/bug4666224.html 8144124  macosx-all
+java/awt/event/MouseEvent/AltGraphModifierTest/AltGraphModifierTest.java 8162380 generic-all
+java/awt/Window/TranslucentJAppletTest/TranslucentJAppletTest.java 8163086 macosx-all
+java/awt/image/multiresolution/MultiResolutionIcon/IconTest.java 8250804 macosx-all,linux-all
```

Comment:
I still think that the sort based on test names is better, but if the current list is sorted by the bugid then the JDK-8250804 is out of order.

Reply:
That's because is *was* under https://bugs.openjdk.java.net/browse/JDK-8163285 but that got closed as a dup of 8250804 .. which seems backwards to me as the new one is usually the dup.
But in any case I am not going to lose any sleep over this one anomaly that it caused.

Reply:
That's why I said it is a bad idea to sort using the bugid, since it is impossible to mantains this even in one review request.